pyartcd/pyartcd/pipelines/build_microshift.py

```diff
@@ -56,36 +58,80 @@ def __init__(self, runtime: Runtime, group: str, assembly: str,
             self._doozer_env_vars["DOOZER_DATA_PATH"] = ocp_build_data_url
 
     async def run(self):
-        self._slack_client.bind_channel(self.group)
-        slack_response = await self._slack_client.say(f":construction: Build microshift for assembly {self.assembly} :construction:")
-        slack_thread = slack_response["message"]["ts"]
-
+        slack_client = None
         try:
             await load_group_config(self.group, self.assembly, env=self._doozer_env_vars)
             releases_config = await load_releases_config(Path(self._doozer_env_vars["DOOZER_WORKING_DIR"], "ocp-build-data"))
             assembly_type = get_assembly_type(releases_config, self.assembly)
             if assembly_type not in self.SUPPORTED_ASSEMBLY_TYPES:
                 raise ValueError(f"Building MicroShift for assembly type {assembly_type.value} is not currently supported.")
-            release_name = get_release_name(assembly_type, self.group, self.assembly, None)
+
+            custom_payloads = None
+            if assembly_type is AssemblyTypes.STREAM:
+                # rebase against nightlies
+                # rpm version-release will be like `4.12.0~test-202201010000.p?`
+                if not self.payloads:
```

Comment:
Maybe I missed something, but shouldn't we check for `self.no_rebase` here?

Reply:
Makes sense. Error shouldn't be raised if no_rebase is set.

Reply:
Ah, it wasn't clear to me that no_rebase was only useful for testing.  I see your comment so we're good